src/main.rs

```diff
@@ -119,7 +120,8 @@ fn main() {
             std::process::exit(exitcode::CONFIG);
         });
 
-        let result = topology::start_validated(config, diff, pieces, opts.require_healthy).await;
+        let result =
+            topology::start_validated(config.clone(), diff, pieces, opts.require_healthy).await;
```

Comment:
Why is this cloned?

Reply:
Good catch, leftover from a previous version.